SmartSaver/forms/finances/goals/ExpandedGoalForm.cs

```
@@ -51,36 +52,19 @@ private void Init()
 
             DisplayTable();
         }
+
+        [SuppressMessage("ReSharper", "PossibleNullReferenceException")]
```

This thing is weird, what is it for?

---

It suppresses the error message about possible exception. I  use it because these columns will definitely be there because of the table constructor. However maybe we should implement some kind of error checking here.